sonic_package_manager/service_creator/feature.py

```diff
@@ -15,6 +16,15 @@
     'set_owner': 'local'
 }
 
+AUTO_TS_GLOBAL = "AUTO_TECHSUPPORT"
+AUTO_TS_FEATURE = "AUTO_TECHSUPPORT_FEATURE"
+CFG_STATE = "state"
+# TODO: Enable available_mem_threshold once the mem_leak_auto_ts feature is available
+DEFAULT_AUTO_TS_FEATURE_CONFIG = {
+    'state': 'disabled',
+#   'available_mem_threshold': '10.0',
```

Comment:
I assume it is commeneted because available_mem_threshold is not yet merged. I suggest to remove the comment since it won't affect anyway.

Reply:
Handled